.github/workflows/update_disabled_tests.yml

```diff
@@ -41,3 +41,9 @@ jobs:
           user_email: "test-infra@pytorch.org"
           user_name: "Pytorch Test Infra"
           commit_message: "Updating condensed disabled tests stats"
+
+      - name: Upload file to s3
+        run: python3 .github/scripts/upload_to_s3.py "disabled-tests-condensed.json"
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Nit: I think if we are using self-hosted runner here, i.e. linux.large, you don't even need to set the key as the runner has read/write access to S3 (I just test it out)

Reply:
theres are run on github provided runners